Microsoft.Toolkit.Win32/Microsoft.Toolkit.Wpf.UI.XamlHost/WindowsXamlHostBase.cs

```diff
@@ -106,15 +141,15 @@ protected override HandleRef BuildWindowCore(HandleRef hwndParent)
         }
 
         /// <summary>
-        /// The default implementation of SetContent applies XamlRootInternal to desktopWindowXamSource.Content.
+        /// The default implementation of SetContent applies ChildInternal to desktopWindowXamSource.Content.
         /// Override this method if that shouldn't be the case.
         /// For example, override if your control should be a child of another WindowsXamlHostBase-based control.
         /// </summary>
-        protected virtual void SetContent()
+        protected virtual void SetContent(Windows.UI.Xaml.UIElement newValue)
```

Comment:
This seems to be causing an issue in the sample app. 

```Severity	Code	Description	Project	File	Line	Suppression State
Error		Unknown build error, 'Cannot resolve dependency to Windows Runtime type 'Windows.UI.Xaml.UIElement'. When using the ReflectionOnly APIs, dependent Windows Runtime assemblies must be resolved on demand through the ReflectionOnlyNamespaceResolve event. Line 16 Position 26.'	Microsoft.Toolkit.Sample.Wpf.App	C:\Users\nikol\source\WindowsCommunityToolkit\Microsoft.Toolkit.Win32\Microsoft.Toolkit.Sample.Wpf.App\MainWindow.xaml	16	
```

Reply:
I've raised this in #2435. 
@ryalanms FYI

Reply:
This is resolved in #2435 and #2437